website/docs/guides/1.17.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,27 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.17.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-1170-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.17.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.17.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.17.0 has a number of new and exciting features.
+
+**New Resources, Data Sources, and Features:**
+
+- You can now Create and Delete [MongoDB Atlas OIDC Workforce identity providers](https://www.mongodb.com/docs/atlas/security-oidc/#configure-oidc-authorization) with `mongodbatlas_federated_settings_identity_provider` [resource](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/federated_settings_identity_provider).
+
+**Breaking Changes:**
+
+- Attribute `audienceClaim` used in OIDC IdPs has been replaced with `audience` attribute in `mongodbatlas_federated_settings_identity_provider` [resource](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/federated_settings_identity_provider) and [`mongodbatlas_federated_settings_identity_provider`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/data-sources/federated_settings_identity_provider) and [`mongodbatlas_federated_settings_identity_providers`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/data-sources/federated_settings_identity_providers) data sources. This change reflects the most recent changes in the Atlas API [Federated Authentication](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Federated-Authentication/operation/updateIdentityProvider)
```

Comment:
q: For a user that has already imported the resource in a previous version, I imagine with this schema change they have to reimport the resource right? We could provide some guidance on what actions need to be taken for each case.

Reply:
Good point! Done.